app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -46,12 +46,17 @@ public enum SettingsEnum {
     ADREMOVER_HIDE_LATEST_POSTS("revanced_adremover_hide_latest_posts", true, ReturnType.BOOLEAN, true),
     ADREMOVER_HIDE_CHANNEL_GUIDELINES("revanced_adremover_hide_channel_guidelines", true, ReturnType.BOOLEAN, true),
     // Buttons
+    HIDE_ACTION_BAR("revanced_action_bar", false, ReturnType.BOOLEAN, false),
     HIDE_LIKE_BUTTON("revanced_like_button", false, ReturnType.BOOLEAN, false),
     HIDE_DISLIKE_BUTTON("revanced_dislike_button", false, ReturnType.BOOLEAN, false),
+    HIDE_SHARE_BUTTON("revanced_share_button", false, ReturnType.BOOLEAN, false),
+    HIDE_LIVE_CHAT_BUTTON("revanced_live_chat_button", false, ReturnType.BOOLEAN, false),
+    HIDE_REPORT_BUTTON("revanced_report_button", false, ReturnType.BOOLEAN, false),
+    HIDE_SHORTS_BUTTON("revanced_create_shorts_button", false, ReturnType.BOOLEAN, false),
+    HIDE_THANKS_BUTTON("revanced_thanks_button", false, ReturnType.BOOLEAN, false),
+    HIDE_CLIP_BUTTON("revanced_create_clip_button", false, ReturnType.BOOLEAN, false),
```

Comment:
To offer this patch, I think, using these defaults is fine. Make sure to apply them to the patch too.
```suggestion
    HIDE_SHORTS_BUTTON("revanced_create_shorts_button", true, ReturnType.BOOLEAN, false),
    HIDE_THANKS_BUTTON("revanced_thanks_button", true, ReturnType.BOOLEAN, false),
    HIDE_CLIP_BUTTON("revanced_create_clip_button", true, ReturnType.BOOLEAN, false),
```